Gemfile.lock

```diff
@@ -16,27 +16,6 @@ GIT
 
 GEM
   remote: https://rails-assets.org/
-  specs:
-    rails-assets-bootstrap-tabcollapse (0.2.7)
-    rails-assets-gemini-scrollbar (1.5.3)
-    rails-assets-intro.js (2.9.3)
-    rails-assets-jquery (3.4.1)
-    rails-assets-jquery-stickit (0.2.14)
-      rails-assets-jquery (>= 1.6)
-    rails-assets-jquery.scrollTo (2.1.2)
-      rails-assets-jquery (>= 1.8)
-    rails-assets-lodash (4.17.15)
-    rails-assets-moment (2.24.0)
-    rails-assets-perfect-scrollbar (0.8.1)
-    rails-assets-readmore (2.2.1)
-      rails-assets-jquery (>= 2.1.4)
-    rails-assets-slick-carousel (1.8.1)
-      rails-assets-jquery (>= 1.7)
-    rails-assets-tether (1.4.7)
-    rails-assets-waypoints (4.0.1)
```

Comment:
@mahesh-krishnakumar I see changes to `Gemfile.lock`, without any changes to `Gemfile`. The current lockfile also looks incorrect - unlike on the `master` branch, it puts a lot of non-asset gems under the  `remote: https://rails-assets.org/`. Restoring the Gemfile from `master` branch should work fine - I can't think of a reason why this file would have changed. 🤔

Reply:
Fixed. Looks like has to do with the wrong `bundler` version I had initially in my system. It does match with the project now.

https://github.com/pupilfirst/pupilfirst/commit/ce468605b060ed96c3ab3e8477bdce3247cadc2e